pkg/kv/kvserver/replica_command.go

```diff
@@ -1645,6 +1645,18 @@ func prepareChangeReplicasTrigger(
 	return crt, nil
 }
 
+func within10s(ctx context.Context, fn func() error) error {
+	retOpts := retry.Options{InitialBackoff: time.Second, MaxBackoff: time.Second, MaxRetries: 10}
+	var err error
+	for re := retry.StartWithCtx(ctx, retOpts); ; re.Next() {
+		err = fn()
+		if err == nil {
+			return nil
+		}
+	}
+	return ctx.Err()
```

Comment:
Huh, the linter says this unreachable. Am I... not seeing something?

Reply:
I mean this code is definitely busted, I should return `err` before ctx.Err() but still